src/Text/Pandoc/Readers/DocBook.hs

```diff
@@ -960,12 +960,22 @@ parseBlock (Elem e) =
                       let aligns = case colspecs of
                                      [] -> replicate numrows AlignDefault
                                      cs -> map toAlignment cs
+                      let parseWidth s = safeRead (T.filter (\x -> (x >= '0' && x <= '9')
```

Comment:
Is this field just a decimal number with no units?  If so, do we need to filter?  If not, are we ignoring the units?

Reply:
It's percent - but the percent sign may or may not be there from what I've seen. I though this was safest - but there may be better ways.